### PR TITLE
Portland 2021 blog post - tickets on sale and CFP reminder

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

**Comment by thatdocslady** [comment]
@plaindocs @mxsasha please turn on any website flags that might need turning on :)

**Comment by mxsasha** [comment]
#1409 should be merged before this one, to set up the grant page

**Comment by mxsasha** [comment]
I enabled the tickets flag, but the tito event is not live yet?

Also, I left the two top buttons to be sponsorship and CFP - if we want tickets there we have to pick one to remove.

File: docs/conf/portland/2021/news/tickets-on-sale-cfp-reminder.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+We’ll be reviewing all the talk proposals during this month, and will be contacting everyone who submitted a proposal by {{ cfp.notification }}.
+
+Grant program
+-------------
+
+We updated our grant program application to take into account the virtual format of the event.
+We're excited to be able to offer free tickets to a larger number of people this year, since there are now minimal other costs to attend the event.
+
+You can apply by **June 5th, 2020, Midnight PST** on `our website <https://www.writethedocs.org/conf/portland/2021/opportunity-grants/>`_.
```
Comment (mxsasha) [inline docs/conf/portland/2021/news/tickets-on-sale-cfp-reminder.rst]:
```suggestion
You can apply by **March 1st, 2021, Midnight PST** on `our website <https://www.writethedocs.org/conf/portland/2021/opportunity-grants/>`_.
```

**Review by mxsasha — APPROVED** [review]

**Comment by thatdocslady** [comment]
@mxsasha the Tito event is in draft state waiting for @ericholscher's review :)

**Review by ericholscher — APPROVED** [review]
Looks good! 🎉

**Pull request merged**